app/models/notification.rb

```diff
@@ -96,32 +96,34 @@ def update_from_api_response(api_response, unarchive: false)
 
   def download_subject
     user.github_client.get(subject_url)
+  rescue Octokit::Forbidden, Octokit::NotFound
   end
 
   def update_subject
     return unless Octobox.config.fetch_subject
+
+    remote_subject = download_subject
+    return unless remote_subject.present?
```

Comment:
Actually I think this should be placed back in the respective clauses; the purpose here was to avoid making a needless API call by the `return if updated_at - subject.updated_at < 2.seconds`, so this is more than a cosmetic change. I'm up for something more elegant if it keeps that functionality though!